src/parse.rs

```diff
@@ -16,3 +16,38 @@ impl Parse for AsyncItem {
         Ok(AsyncItem(item))
     }
 }
+
+#[derive(Default)]
```

Comment:
Instead of deriving default here, maybe give an explicit impl where send_bound is true (given that that is the default behaviour)

Reply:
Good point. Fixed.